app/pages/IdentityIssuance/PickProvider/PickProvider.tsx

```diff
@@ -220,6 +220,7 @@ export default function IdentityIssuanceChooseProvider({
                     ledgerCallback={withLedger}
                     preCallback={getIdentityNumber}
                     disabled={!provider}
+                    identityVersion={0}
```

Comment:
I feel like I have seen this before. Hardcoding the version seems like a bad idea.

Reply:
I have explained the hardcoding now.

Reply:
I think the better way to do this would be to change the name of the parameter. If the parameter was an enum then you could have it as:
```
Seed = 0,
Key = 1
```
and that way you would supply it like:
```
exportType={Seed}
```
making it very clear what is happening. It is a little hacky to call the input `identityVersion` when in reality the function doesn't care about the identity, it just has to know whether to get the seed or the key.